src/rules/noMagicNumbersRule.ts

```diff
@@ -17,10 +17,20 @@
 
 import * as ts from "typescript";
 
-import { isCallExpression, isIdentifier } from "tsutils";
+import {
+    forEachComment,
+    isCallExpression,
+    isElementAccessExpression,
+    isIdentifier,
+    isSameLine } from "tsutils";
 import * as Lint from "../index";
 import { isNegativeNumberLiteral } from "../language/utils";
 
+interface Options {
+    allowedNumbers: number[] | Set<string>;
```

Comment:
`Set<string>` does not make sense for numbers. why is that here and can it be just `number[]` (or `Set<number>`)?